Resources/Prototypes/_Mono/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```
@@ -0,0 +1,61 @@
+  - type: GunWieldBonus
+    minAngle: -21
+    maxAngle: -46.5
+  - type: Gun
+    minAngle: 21
+    maxAngle: 48
+    angleIncrease: 6
+    angleDecay: 24
+    fireRate: 5 # 300 rpm
+    selectedMode: FullAuto
+    availableModes:
+    - SemiAuto
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)